src/tests/ftest/soak/soak.yaml

```diff
@@ -100,31 +108,31 @@ jobsmoke:
 # 1, 16 and 32 tasks per node
 job_ior_daos:
     name: job_ior_daos
-    time: 10
+    time: 20
     tasks:
         - 1
         - 16
-        - 32
+        #- 32
```

Comment:
Is this intentionally left commented?

Reply:
I need to open a ticket for this.   I can not get 32 task per node to run

Reply:
Ideally we want to handle this with `cancelForTicket("DAOS-XXXX")` in the code after you have read the param from the yaml.  It will make it easier to find bugs that are holding up testing.